src/main/java/org/takes/facets/fallback/FbStatus.java

```diff
@@ -45,7 +47,46 @@
  */
 @EqualsAndHashCode(callSuper = true)
 public final class FbStatus extends FbWrap {
-
+    /**
+     * Ctor.
+     * @param code HTTP status code
+     * @since 0.16.10
+     */
+    public FbStatus(final int code) {
+        this(new Condition<Integer>() {
+            @Override
+            public boolean fits(final Integer status) {
+                return code == status;
+            }
+        });
+    }
+    /**
+     * Ctor.
+     * @param check HTTP status code predicate
+     * @since 0.16.10
+     */
+    public FbStatus(final Condition<Integer> check) {
+        this(check, new Fallback() {
+            @Override
+            public Iterator<Response> route(final RqFallback req)
+                throws IOException {
+                final Response res = new RsWithStatus(req.code());
+                final String message = res
+                    .head().iterator().next().substring("HTTP 1.1 ".length());
```

Comment:
@ggajos in that case `200 OK` will be extracted, and the full message in the body will be something like `200 OK: <message from throwable>`. I assume that's exactly what we need, right? If someone creates fallback for status 200, then probably that's what he or she wants. Or do you mean there's a typo in a string, i.e. it should be `HTTP/1.1` instead of `HTTP 1.1`?